sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/TestUtils.java

```diff
@@ -51,36 +51,35 @@
  * Contains helper methods for generating inputs for test methods
  */
 final class TestUtils {
-
-    static final String VALID_MODEL_ID = "95537f1b-aac4-4da8-8292-f1b93ac4c8f8";
-    static final String LABELED_MODEL_ID = "a0a3998a-b3c0-4075-aa6b-c4c4affe66b7";
     static final String INVALID_MODEL_ID = "a0a3998a-4c4affe66b7";
     static final String INVALID_STATUS_MODEL_ID = "22138c4e-c4b0-4901-a0e1-6c5beb73fc1d";
```

Comment:
these two seems valid model id and status model id?

Reply:
>     static final String INVALID_MODEL_ID = "a0a3998a-4c4affe66b7";
It is not a valid UUID format for model Id.

>     static final String INVALID_STATUS_MODEL_ID = "22138c4e-c4b0-4901-a0e1-6c5beb73fc1d";
And this one has an invalid status.